rero_ils/modules/ill_requests/forms.py

```diff
@@ -15,7 +15,7 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-"""Forms definitions about ILL request in public view."""
+"""Forms definitions about ILLrequest in public view."""
```

Comment:
I am not sure what this is, but have made all of them "ILLrequest". Hopefully a translatable name could be found?

Reply:
I think *ILL request* is better. It stands for InterLibrary Loan request.